### ISSUE TITLE
Files GENERIC*.make point to non-existing files

### ISSUE THREAD

**Issue by hjjvandam** [JorgeStolfi/JSLIBS#7]
Dear Jorge,

I am reviewing artifacts for the Super Computing 2024 conference. While trying to reproduce one of the artifacts submitted with a paper I have found that the authors have used your SPMAT library. For that reason I am having to install this library in the Chameleon Cloud that the conference organizers have made available. 

The issue I am hitting is that if I run:

- git clone https://github.com/JorgeStolfi/JSLIBS
- cd JSLIBS
- make

I get:
```
= = all-build = = = = = = = = = = = = = = = = = = = = = = = = = = = = =
libjs
make[1]: Entering directory '/home/cc/JSLIBS/libjs'
Makefile:9: /programs/c/GENERIC-LIB.make: No such file or directory
make[1]: *** No rule to make target '/programs/c/GENERIC-LIB.make'.  Stop.
make[1]: Leaving directory '/home/cc/JSLIBS/libjs'
```
The Makefile in `libjs` reads:
```
# Last edited on 2023-02-06 11:34:29 by stolfi
  
LIBNAME = libjs

# OTHER_C_FLAGS := -pg

IGNORE :=

include ${STOLFIHOME}/programs/c/GENERIC-LIB.make
```
Obviously the include statement points to the wrong location. I can easily fix this if I have GENERIC-LIB.make. This file is present in the directory JSLIBS but it is simply a soft link to a file I don't have. I.e.:
```
$ ls -l JSLIBS/GENERIC* 
lrwxrwxrwx 1 cc cc 24 Jul 31 22:22 JSLIBS/GENERIC-LIB-TEST.make -> ../GENERIC-LIB-TEST.make
lrwxrwxrwx 1 cc cc 19 Jul 31 22:22 JSLIBS/GENERIC-LIB.make -> ../GENERIC-LIB.make
lrwxrwxrwx 1 cc cc 24 Jul 31 22:22 JSLIBS/GENERIC-ROOT-DIR.make -> ../GENERIC-ROOT-DIR.make
lrwxrwxrwx 1 cc cc 15 Jul 31 22:22 JSLIBS/GENERIC.make -> ../GENERIC.make
```
In other words, when you committed these `GENERIC*.make` files git checked the soft links in instead of the contents of these files. Would you be in a position to fix this issue, please?

Best wishes,

Hubertus van Dam
Brookhaven National Laboratory
